library/include/ublox_structs.h

```diff
@@ -43,6 +43,7 @@ struct FixType {
         last = Undefined
     };
     static constexpr std::array<const char*, last + 1> name { "No Fix", "Dead Reck.", "2D-Fix", "3D-Fix", "GPS+Dead Reck.", "Time Fix", " N/A" };
+    static constexpr std::array<const char*, last + 1> name_log_safe { "NoFix", "DeadReck", "2D", "3D", "GPS+DeadReck", "TimeFix", "N/A" };
```

Comment:
Pls. replace the old array with the new one instead of adding it

Reply:
The original array is still used in the GUI, where human readability is more important.
Should it still be removed entirely?

Reply:
I would suggest so. The new strings are sufficiently self-explanatory to be shown in user interface contexts.